doc/classes/AudioEffectHardLimiter.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectHardLimiter" inherits="AudioEffect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Adds a hard limiter audio effect to an Audio bus.
+	</brief_description>
+	<description>
+		A limiter is an effect designed to disallow sound from going over a given dB threshold. Hard limiters predict volume peaks, and will smoothly apply gain reduction when a peak crosses the ceiling threshold to prevent clipping and distortion. It preserves the waveform and prevents it from crossing the ceiling threshold. Adding one in the Master bus is always recommended as a safety measure to prevent sudden volume peaks from occurring, and to prevent distortion caused by clipping.
+	</description>
+	<tutorials>
+		<link title="Audio buses">$DOCS_URL/tutorials/audio/audio_buses.html</link>
+	</tutorials>
+	<members>
+		<member name="ceiling_db" type="float" setter="set_ceiling_db" getter="get_ceiling_db" default="-0.3">
+			The waveform's maximum allowed value, in decibels. Value can range from -24 to 0.0.
+			The default value of -0.3 dB is set to prevent potential inter sample peaks (ISP) from crossing over 0dB, which can cause slight distortion on some older hardware.
```

Comment:
It feels like there's many ways to say it, but "inter-sample" feels the most appropriate to me.
```suggestion
			The waveform's maximum allowed value, in decibels. This value can range from [code]-24.0[/code] to [code]0.0[/code].
			The default value of [code]-0.3[/code] prevents potential inter-sample peaks (ISP) from crossing over 0 dB, which can cause slight distortion on some older hardware.
```